code/modules/overmap/ships/computers/shuttle.dm

```diff
@@ -63,6 +70,24 @@
 			shuttle.set_destination(possible_d[D])
 		return TRUE
 
+	if(action == "jump_to_overmap" && jump_to_overmap_capable)
+		// get our shuttle
+		var/obj/effect/overmap/visitable/ship/landable/landable = connected
+
+		// get horizon (or whatever other main map)
+		var/obj/effect/overmap/visitable/main = GLOB.map_sectors["1"] ? GLOB.map_sectors["1"] : GLOB.map_sectors[GLOB.map_sectors[1]]
```

Comment:
i think you should be able to get the main zlevels from the current_map directly without having to hardcode it as "1"

Reply:
done